### PR TITLE
Master

### PR DESCRIPTION
**Pull request by theCyanEYED**
Added : gpg4win 2.2.5, qbittorrent 3.2.3, teamspeak 3.0.17

### CODE REVIEW

File: just-install.json [hunk 1]
```diff
@@ -140,6 +140,15 @@
...
     },
+    
+    "defraggler": {
+      "version": "2.1.9.982",
+      "installer": {
+        "kind": "nsis",
+        "x86": "http://download.piriform.com/dfsetup219.exe"
+      }
+    },
+
```
Comment (lvillani) [inline just-install.json]: Please remove the blank line.

File: just-install.json [hunk 2]
```diff
@@ -140,6 +140,15 @@
         }
       }
     },
+
```
Comment (lvillani) [inline just-install.json]: Please remove the blank line.

File: just-install.json [hunk 3]
```diff
@@ -773,6 +782,13 @@
         }
       }
     },
+    "windirstat": {
+      "version": "1.1.2",
+      "installer": {
+        "kind": "nsis",
+        "x86": "http://downloads.sourceforge.net/project/windirstat/windirstat/1.1.2%20installer%20re-release%20%28more%20languages%21%29/windirstat1_1_2_setup.exe?r=&ts=1441302161&use_mirror=heanet",
```
Comment (lvillani) [inline just-install.json]: Extra comma at the end of the line.

**Comment by lvillani** [comment]
I guess it was closed by accident? The PR looks fine now.

**Comment by theCyanEYED** [comment]
Pull request sent with the wrong comment ?
Oh well. I'll update it.

**Pull request merged**